api/src/main/java/org/openmrs/module/aijar/dataintegrity/InvalidTBEncounters.java

```diff
@@ -98,14 +98,14 @@
 		for (Obs obs : obsList) {
 			Patient patient = obs.getEncounter().getPatient();
 			RuleResult<Patient> ruleResult = new RuleResult<>();
-			ruleResult.setActionUrl("coreapps/patientdashboard/patientDashboard.page?patientId=" + patient.getUuid());
-			ruleResult.setNotes("The District TB Number " + getTbNumber(patient, obs.getEncounter()) + " is used by another patient");
+			ruleResult.setActionUrl("htmlformentryui/htmlform/editHtmlFormWithStandardUi.page?patientId=" + patient.getUuid() + "&encounterId=" + obs.getEncounter().getId());
```

Comment:
@ssmusoke The formUuid is not necessary when editing an existing entry. It's only required when making a new entry